CV.html

```diff
@@ -1,69 +0,0 @@
-<!DOCTYPE html>
-<html lang="pl">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <title>Portfolio</title>
-     <meta name="description" content="Interaktywny poradnik szybkiego startu dla Brackets.">
-        <link rel="stylesheet" href="portfolio.css">
-        <link href="https://fonts.googleapis.com/css?family=Pacifico|Poiret+One" rel="stylesheet">
-        
-               <script type="text/javascript">
-            
-            function odliczanie()
-            {
-                var data = new Date();
-                
-                var godzina = data.getHours();
-                if(godzina <10)
-                    {
-                        godzina="0"+godzina;
-                    }
-                var minuta = data.getMinutes();
-                if(minuta <10)
-                    {
-                        minuta="0"+minuta;
-                    }
-                var sekunda = data.getSeconds();
-                if(sekunda <10)
-                    {
-                        sekunda="0"+sekunda;
-                    }
-                
-                document.getElementById("zegar").innerHTML = godzina+":"+minuta+":"+sekunda;
-                
-                setTimeout("odliczanie()",1000);
-                
-            }
-    
-        
-        </script>
-        
-</head>
-<body onload="odliczanie();">
-   <div id="logo">
-       <div id="imie">
```

Comment:
If you make ID tags, theres no need to nest another ID inside, classes are more flexible